cmd/list.go

```diff
@@ -51,9 +53,64 @@ func RunList() *cobra.Command {
 				os.Exit(1)
 			}
 
-			fmt.Println(torrents)
+			PrintList(torrents)
 		}
 	}
 
 	return command
 }
+
+func PrintList(torrents []qbittorrent.Torrent) {
+	for _, torrent := range torrents {
+		fmt.Printf("%-60s\t[%s]\n", torrent.Name, torrent.State)
```

Comment:
Maybe add `status` before state? Only a minor suggested improvement.

Reply:
I am assuming you meant to literally add a "Status: " header, since there is no field `status` field on the Torrent struct, since State is essentially status.